.gitbook/source/spec-relations.dot

```diff
@@ -26,6 +29,8 @@ digraph D {
         RFC7662_EXT [label = "RFC7662\nOAuth 2.0 Token Introspection", href = "https://tools.ietf.org/html/rfc7662", fillcolor = azure2, style = filled]
         RFC7523_EXT [label = "RFC7523\nJSON Web Token (JWT) Profile\nfor OAuth 2.0 Client Authentication and Authorization Grants", href = "https://tools.ietf.org/html/rfc7523", fillcolor = azure2, style = filled,]
         RFC7515_EXT [label = "RFC7515\nJSON Web Signature (JWS)", href = "https://tools.ietf.org/html/rfc7515", fillcolor = azure2, style = filled]
+        DID_EXT [label = "W3C DID\nDecentralized Identifiers (DIDs) v1.0", href = "https://www.w3.org/TR/did-core/", fillcolor = azure2, style = filled]
+        VC_EXT  [label = "W3C VC\nVerifiable Credentials Data Model v1.1", href = "https://www.w3.org/TR/vc-data-model/", fillcolor = azure2, style = filled]
```

Comment:
labels mention a minor version while the links point to v1.x

Reply:
The site clearly states V1.1?
<img width="692" alt="Screenshot 2022-09-13 at 11 57 28" src="https://user-images.githubusercontent.com/703298/189872333-e0da347b-293f-4c9b-a23f-e14c5079e1a2.png">

Reply:
https://www.w3.org/TR/2022/REC-vc-data-model-20220303/ is the permalink of (the current version of) v1.1

We now link https://www.w3.org/TR/vc-data-model/ that currently points to v1.1, but used to point to v1.0 and maybe to v1.2 in the future?
There is also a v2 in the making on https://www.w3.org/TR/vc-data-model-2.0/

Reply:
Then we use the permalink